collector/src/main/java/com/bitmovin/analytics/utils/Util.java

```diff
@@ -23,6 +23,8 @@
     public static final String SMOOTH_STREAM_FORMAT = "smooth";
     public static final int MILLISECONDS_IN_SECONDS = 1000;
     public static final int VIDEOSTART_TIMEOUT = 1000 * 60; // in milliseconds
+    public static final int ANALYTICS_QUALITY_CHANGE_COUNT_THRESHOLD = 50;
+    public static final int ANALYTICS_QUALITY_CHANGE_COUNT_RESET_INTERVAL = 60 * 60 * 1000;
```

Comment:
Please add a comment which unit is used (milliseconds, seconds). Makes it clearer later on what this should be.